### DISCUSSION TITLE
signInWithProvider without web

### DISCUSSION THREAD

**Discussion by shinriyo** [supabase/supabase#3638] [Feature Requests]
```
import 'package:supabase/supabase.dart';
```

```
    final response = await Supabase.instance.client.auth.signInWithProvider(
      Provider.facebook,
      options: AuthOptions(
          redirectTo: kIsWeb ? null : 'io.supabase.flutter://reset-callback/'),
    )
```

it is my code.

but, it redirects web site.
I want to in the app without redirect to website.

Can it do it?

**Comment by dshukertjr** [comment]
Hi @shinriyo!

The Supabase team is working on bringing native sign in on mobile apps at the moment. You can read more about it on [this issue](https://github.com/supabase/gotrue-dart/issues/27).
